plugins/wdc/wdc-nvme.c

```diff
@@ -4071,10 +4066,14 @@ static void wdc_print_fw_act_history_log_normal(__u8 *data, int num_entries, __u
 				printf("%s", time_str);
 				printf("     ");
 			} else {
-				printf("   ");
```

Comment:
Same comment as above - need to leave this code in to support FW that uses the Timestamp instead of Power on Hour.

Reply:
Addressed.